Add readme for react-embed example

lizraeli opened the pull request:
Adds a readme explaining how to set up the example project for `react-embed`.

fzembow commented:
@lizraeli does `parcel-bundler` really need to be installed as a global dependency?

seems to me that you could just have it be a dev dependency and then `npm run dev` would be able to use it?

lizraeli commented:
> @lizraeli does `parcel-bundler` really need to be installed as a global dependency?
> 
> seems to me that you could just have it be a dev dependency and then `npm run dev` would be able to use it?

@fzembow good point. Added parcel as a local dev-dependency and updated the readme.
